frontend/public/components/utils/async.tsx

```diff
@@ -23,7 +16,7 @@ export class AsyncComponent extends React.Component<AsyncComponentProps, AsyncCo
   private isAsyncMounted: boolean = false;
 
   static getDerivedStateFromProps(props, state) {
-    if (!sameLoader(props.loader)(state.loader)) {
+    if (!_.isEqual(props.loader, state.loader)) {
```

Comment:
It looks like this uses strict equality for functions if I'm reading the lodash docs right. Is this the same as using `===` in that case?

https://lodash.com/docs/4.17.15#isEqual

Reply:
@spadgett yes, but in this case you are comparing the functions not strings, right ?

Reply:
ooh, right :( 

Well in that case I dont have a simple solution for the issue

Reply:
> @spadgett yes, but in this case you are comparing the functions not strings, right ?

Yeah, but if it's an inline function, it will always evaluate as changed and reload the component.